.gitlab-ci.yml

```diff
@@ -77,7 +77,7 @@ publish:
       do {
           try {
               # The grants option at the end is used to allow public access on the files we upload as the acls only aren't enough.
-              aws s3 cp artifacts-out/ s3://dd-windowsfilter/builds/tracer/${CI_COMMIT_SHA} --recursive --region us-east-1 --exclude "*" --include "*.zip" --include "*.msi" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+              aws s3 cp artifacts-out/ s3://dd-windowsfilter/builds/tracer/${CI_COMMIT_SHA} --recursive --region us-east-1 --exclude "*" --include "*.zip" --include "*.msi" --include "telemetry_forwarder.exe" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
```

Comment:
What is `telemetry_forwarder.exe`? Just curious.

Reply:
It's a Go binary built by the SSI injection platform team that we call as part of the native loader. This then forwards details about the injection success/failure to the instrumentation-telemetry backend, and then drives the troubleshooting experience for us and customers. It is shipped as part of the injection platform stuff, and is basically a way for us to use the telemetry infra without having to implement that all in C 😄